lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -159,6 +160,52 @@ sub edit_banner : Path('/admin/banner/edit') Args(0) RequireAuth(banner_editor)
     }
 }
 
+sub email_search : Path('/admin/email-search') Args(0) RequireAuth(account_admin) HiddenOnSlaves {
+    my ($self, $c) = @_;
+
+    my $form = $c->form(form => 'Admin::EmailSearch');
+    my @results;
+
+    if ($c->form_submitted_and_valid($form)) {
+        try {
+            @results = $c->model('Editor')->search_by_email(
+                $form->field('email')->value // '',
+            );
+        } catch {
+            my $error = $_;
+            if ("$error" =~ m/invalid regular expression/) {
+                $form->field('email')->add_error(l('Invalid regular expression.'));
```

Comment:
This string already existed in root/static/scripts/release-editor/trackParser.js